From dfd0b86cc571daf0f5f72a17d4eaf47fece8aefc Mon Sep 17 00:00:00 2001
From: cjaneway <72281149+cjaneway@users.noreply.github.com>
Date: Sat, 31 Oct 2020 22:14:02 -0700
Subject: [PATCH 19/40] Update INSTALL

Practical installation of pkgs. needed to build the program on Ubuntu/Mint.
---
 INSTALL | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/INSTALL b/INSTALL
index 8041fc4d..52879eac 100644
--- a/INSTALL
+++ b/INSTALL
@@ -12,6 +12,10 @@ Build depends:
     QtSql compiled with sqlite support
     QtWebKit
     QtXml
+    
+ Linux Mint 20 (others?):
+ sudo apt install qt5-default qttools5-dev-tools qttools5-dev
+ sudo apt-get install qtmultimedia5-dev libqt5multimediawidgets5 libqt5multimedia5-plugins libqt5multimedia5
 
 
 Instructions for Linux and other UNIX-like systems:
-- 
2.43.0

